src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,10 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <title>index page</title>
+</head>
+<body>
+<h1>Information written to our resource</h1>
+
```

Comment:
remove the empty line. and make code like:
<html>
      <head>
             <title>index page</title>
      </head>
      <body>
              <h1>Information written to our resource</h1>
      </body>
</html>